src/components/Stepper.js

```diff
@@ -0,0 +1,32 @@
+// Stepper.js
+import React from "react";
+
+export default function Stepper({ steps, activeStep }) {
```

Comment:
Add a comment explaining what this stepper function does

Reply:
Updated with comment